bedrock/firefox/templates/firefox/whatsnew/index-account.html

```diff
@@ -36,22 +41,52 @@
   </header>
 
   <section class="content-main">
-    <div class="mzp-l-content">
+    <div class="mzp-l-content mzp-t-content-md">
       <div class="mzp-c-emphasis-box">
-        <img class="c-emphasis-box-logo" src="{{ static('protocol/img/logos/firefox/logo-md.png') }}" alt="">
-        <h2 class="c-emphasis-box-title">
-          {{ ftl('whatsnew-account-main-headline') }}
-        </h2>
-        <p>{{ ftl('whatsnew-account-main-lead-in') }}</p>
-        <p class="cta">
-          {{ fxa_button(
-            entrypoint=entrypoint,
-            button_text=ftl('whatsnew-account-main-button'),
-            class_name='mzp-t-small',
-            optional_parameters={'utm_campaign': campaign},
-            optional_attributes={'data-cta-text': 'Sign In', 'data-cta-type': 'fxa-sync', 'data-cta-position': 'primary'}
-          ) }}
-        </p>
+
+      {# If user is signed out (or undetermined), promote account #}
+        <div class="show-fxa-supported-signed-out">
+          <img class="c-emphasis-box-logo" src="{{ static('protocol/img/logos/firefox/logo-md.png') }}" alt="">
+          <h2 class="c-emphasis-box-title">
+            {{ ftl('whatsnew-account-main-headline') }}
+          </h2>
+          <p>{{ ftl('whatsnew-account-main-lead-in') }}</p>
+          <p class="cta">
+            {{ fxa_button(
+              entrypoint=entrypoint,
+              button_text=ftl('whatsnew-account-main-button'),
+              class_name='mzp-t-small',
+              optional_parameters={'utm_campaign': campaign},
+              optional_attributes={'data-cta-text': 'Sign In', 'data-cta-type': 'fxa-sync', 'data-cta-position': 'primary'}
+            ) }}
+          </p>
+        </div>
+
+      {# If the user is signed in, promote mobile #}
+        <div class="show-fxa-supported-signed-in">
+          <img class="c-emphasis-box-logo" src="{{ static('protocol/img/logos/firefox/browser/logo-md.png') }}" alt="">
+        {# If user is in a locale with translated basket messages... #}
+        {% if show_send_to_device %}
+          <h2 class="c-emphasis-box-title">{{ ftl('whatsnew-account-signed-in-headline') }}</h2>
```

Comment:
Could we fall back to the default S2D title here, if a locale doesn't have it translated?